### PR TITLE
render posts on news page

### PR DESCRIPTION
**Pull request by mhasilau**

### CODE REVIEW

File: src/components/news/news.js [hunk 1]
```diff
@@ -32,15 +34,14 @@ export const renderPostsNews = async () => {
     infoDate.className = 'renger-posts-info';
 
     if (user.uuid !== LocalStorageClass.getUID()) {
-      postPlace.style.display = 'none';
-    } else postPlace.style.display = 'block';
+      postPlace.style.display = 'block';
+    } else postPlace.style.display = 'none';
```
Comment (AliakseiKliashchou) [inline src/components/news/news.js:38]: ternary

File: src/home/home.js [hunk 2]
```diff
@@ -19,9 +19,10 @@ export const postForm = () => {
   post_form.addEventListener('submit', event => {
     event.preventDefault();
     post.content = post_content.value;
-    createPost(post)
+    console.log(post);
```
Comment (AliakseiKliashchou) [inline src/home/home.js:22]: log

File: src/home/home.js [hunk 3]
```diff
@@ -19,9 +19,10 @@ export const postForm = () => {
...
+    createPost(post);
     post_content.value = null;
-    window.location.reload();
+    // window.location.reload();
```
Comment (AliakseiKliashchou) [inline src/home/home.js:25]: comment

File: src/components/news/news.js (hunk 1 above)
Comment (mhasilau) [reply src/components/news/news.js:38]: fix

**Pull request merged**